safe/impact_functions/generic/classified_hazard_building.py

```diff
@@ -106,54 +106,55 @@ def get_metadata():
     # Function documentation
     target_field = 'DAMAGED'
     affected_field = 'affected'
-    title = tr('Be impacted by each category')
+    title = tr('Be impacted by each hazard class')
     synopsis = tr(
-        'To assess the impacts of categorized hazard in raster '
-        'format on structure/building raster layer.')
+        'To assess the impacts of classified hazards in raster format on '
+        'building vector layer.')
     actions = tr(
-        'Provide details about how many building would likely need '
-        'to be affected for each category.')
+        'Provide details about how many building would likely be impacted for '
+        'each hazard class.')
     hazard_input = tr(
-        'A hazard raster layer where each cell represents '
-        'the category of the hazard. There should be 3 '
-        'categories: 1, 2, and 3.')
+        'A hazard raster layer where each cell represents the class of the '
+        'hazard. There should be 3 classes: e.g. 1, 2, and 3.')
     exposure_input = tr(
-        'Vector polygon layer which can be extracted from OSM '
-        'where each polygon represents the footprint of a building.')
+        'Vector polygon layer which can be extracted from OSM where each '
+        'polygon represents the footprint of a building.')
     output = tr(
-        'Map of structure exposed to high category and a table with '
-        'number of structure in each category')
+        'Map of structure exposed to high class and a table with number of '
```

Comment:
The impact layer will contain all structures that were exposed to the highest class (3) and a summary table containing the number of structures in each class.